src/manifold/src/face_op.cpp

```diff
@@ -130,6 +148,22 @@ void Manifold::Impl::Face2Tri(const VecDH<int>& faceEdge,
       }
     }
   }
+
+  std::sort(slowTasks.begin(), slowTasks.end(),
+            [](const auto& p1, const auto& p2) { return p1.first < p2.first; });
+  for (auto& task : slowTasks) {
+    task.second.wait();
+    const auto pair = task.second.get();
```

Comment:
`auto` is great for complex types, but it might be nice to be explicit here for readability.